model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -316,7 +316,7 @@ message ProcessBundleProgressRequest {
 // ProcessBundleResponse reporting.
 message MonitoringInfoSpec {
   string urn = 1;
-  string type_urn = 2;
+  repeated string type_urns = 2;
```

Comment:
What does it mean for this to be repeated? A metric only has a single type.

Reply:
I'll add a comment, but this is a list of *allowed* types for a URN conforming to a given `MonitoringInfoSpec`.

As part of expanding the `USER_COUNTER` category to handle distributions and gauges, I need to either expand validation to support those types (done in this PR [here](https://github.com/apache/beam/pull/7868/files#diff-90a5bbb6f36b2c93562e3cffa14350fdR64)), or introduce new user-metric URN-categories for those new types (as discussed below).

Reply:
I removed this change, 👍🏻